application.py

```diff
@@ -23,4 +23,10 @@
 
 @app.route("/")
 def index():
-    return "Project 1: TODO"
+    return render_template("Registration.html")
+@app.route("/Details",methods=["POST"])
```

Comment:
change the route to /register and see that the function can handle both GET and POST methods

Reply:
updated GET and POST method and also attached screenshot in the issue

Reply:
okay work on next assignment.